src/include/switch_rtp.h

```diff
@@ -50,6 +50,7 @@ SWITCH_BEGIN_EXTERN_C
 #define SWITCH_RTP_CRYPTO_KEY_80 "AES_CM_128_HMAC_SHA1_80"
 
 #define SWITCH_RTP_BUNDLE_INTERNAL_PT 21
+#define TELNYX_RTP_DEFAULT_POLL_TIMEOUT_S 2
```

Comment:
We should still use the current default to 5 seconds.

Reply:
Nope, we should use 2 seconds to match Kamailio's fr_timer timout.